app/index.js

```diff
@@ -107,6 +111,19 @@ var NodeModuleGenerator = yeoman.Base.extend({
                 type: 'input',
                 name: 'username',
                 message: 'What is your GitHub username?',
+                store: true,
+                validate: function(input) {
+                    var res =   !input.startsWith('_')   &&
```

Comment:
That's true, but we are already using regexp on line 119, so it is not much of a stretch. But actually it would be even better to just use this package instead: https://github.com/regexhq/regex-username
(sorry I didn't mention this before, didn't know one existed until I looked)